docs/2_2_common_mechanisms.adoc

```diff
@@ -830,7 +830,7 @@ where:
 . latexmath:[\mathbf{f}_{disc}] denotes the state transition function and
 . latexmath:[\mathbf{f}_{event}] denotes the output function of the <<model-partition>> of latexmath:[\mathbf{k}].
 
-_[Such a model partition may represent a task or an interrupt service routine of an embedded system, or a discretized part of a plant model._
+_[Such a model partition may represent a <<task>> or an interrupt service routine of an embedded system, or a discretized part of a plant model._
```

Comment:
I suggest to erase "task" here because this is in particular an example where "task" is intentionally used in a vague way . But being this vague does not result in a benefit for the standard. The ISR and "discretized part of a plant model" may be sufficient as examples. The use of the term task here and the current state of the definition in the glossary even contradicts "A task can be thought of as a simple interface of the scheduler that allows to connect the FMU interface ..."  **What do you think?** -> "Such a model partition may represent an interrupt service routine of an embedded system, or a discretized part of a plant model. "

Reply:
We should at least mention control code, I will produce a "neutral" proposal.

Reply:
Reads now: "Such a model partition may represent part of control code or an interrupt service routine of an embedded system, or a discretized part of a plant model."